genomeNexusPublicApiClient/src/main/java/org/genome_nexus/ApiClient.java

```diff
@@ -52,7 +52,7 @@
 
 public class ApiClient {
 
-    private String basePath = "http://genomenexus.org";
+    private String basePath = "http://localhost:8888";
```

Comment:
Not sure if it is a good idea to have `http://localhost:8888` as the default base path. Maybe we should generate this file from the actual server after we merge and deploy genome-nexus/genome-nexus/pull/449?

Reply:
In the past I have myself "hacked" the PR to replace the api reference here so that it does not show a change. But I think your suggest is better @onursumer ... that we update it again from the production download (no hacking)

Reply:
oh yeah, I forgot about about the base path. I can make that update soon